ui/firestore.rules

```diff
@@ -0,0 +1,9 @@
+rules_version = '2';
+service cloud.firestore {
```

Comment:
Ig this file needs to be updated in future ?

Reply:
yes, it will be updated. Rn, we are only using it for testing.